Microsoft.Toolkit.Uwp.UI/Triggers/IsNullOrEmptyStateTrigger.cs

```diff
@@ -28,22 +29,47 @@ public object Value
         /// Identifies the <see cref="Value"/> DependencyProperty
         /// </summary>
         public static readonly DependencyProperty ValueProperty =
-            DependencyProperty.Register(nameof(Value), typeof(object), typeof(IsNullOrEmptyStateTrigger), new PropertyMetadata(true, OnValuePropertyChanged));
+            DependencyProperty.Register(nameof(Value), typeof(object), typeof(IsNullOrEmptyStateTrigger), new PropertyMetadata(null, OnValuePropertyChanged));
```

Comment:
Why is the default value 'true' ? This is an object property that you bind the object to, so `null` would be the logical default.

Reply:
You'll need to ask the original author(s) of this trigger why the default value was set to `true`.  This PR is changing the default to `null`.

Reply:
D'oh! I read it backwards :-) As for the original author... errr.... that's me :D